### PR TITLE
fix: CLI alias on DBT

### PR DESCRIPTION
**Pull request by rephus**
Closes:  #3505

## How to replicate

- Add alias
Screenshot from 2022-10-21 10-03-41

- DBT run 
Screenshot from 2022-10-21 10-03-32

- lightdash generate
Screenshot from 2022-10-21 10-03-22

- lightdash preview 

Screenshot from 2022-10-21 10-02-56

### CODE REVIEW

File: packages/cli/src/dbt/models.ts [hunk 1]
```diff
@@ -280,7 +280,12 @@ export const getModelsFromManifest = (
                 model.config?.materialized &&
                 model.config.materialized !== 'ephemeral',
         )
-        .map((model) => normaliseModelDatabase(model, adapterType));
+        .map((model) =>
+            normaliseModelDatabase(
+                { ...model, name: model.alias },
```
Comment (IrakliJani) [inline packages/cli/src/dbt/models.ts:285]: @rephus this should be model.name || model.alias, no?

Comment (IrakliJani) [reply packages/cli/src/dbt/models.ts]: ah, interesting. yeah let's have it just in case

**Review by IrakliJani — APPROVED** [review]

**Comment by IrakliJani** [comment]
@rephus have one more question actually. what query does it generate? and do we need to translate back the alias before running that query?

**Comment by rephus** [comment]
> @rephus have one more question actually. what query does it generate? and do we need to translate back the alias before running that query?

Yeah, great question. I created another alias `test_datetimes` this is the request

image

DBT creates the aliased table on DB 

image

**Pull request merged**